### ISSUE TITLE
Support dev packages in Puli CLI

### ISSUE THREAD

**Issue by webmozart** [puli/issues#82]
Depends on: #81

dev packages should be supported in the Puli CLI:

*  a "--dev" option should be added to the "package --add" command that installs a package in dev mode
* dev packages should be separated from non-dev packages in the output of "package --list"

```
Enabled packages:

    vendor/package1 spock packages/package1
    vendor/package2 spock packages/package2

Enabled packages (dev):

    vendor/package3 spock packages/package3
```

* the "%dev%" variable should be added to the "--format" option of the "package --list" command that prints true/false for dev/non-dev packages

**Comment by webmozart** [comment]
@niklongstone Do you want to work on this? :)

**Comment by niklongstone** [comment]
I am on it @webmozart :-)

**Comment by webmozart** [comment]
Implemented in 7d8d05d2f5b35297e3c3f4d5d29a694dc7618be9 and 8706397c35fe2edfdd21b39862775335986d5494.
